src/m365/spo/commands/folder/folder-add.ts

```diff
@@ -45,14 +56,35 @@ class SpoFolderAddCommand extends SpoCommand {
       },
       {
         option: '-n, --name <name>'
+      },
+      {
+        option: '--color [color]'
       }
     );
   }
 
   #initValidators(): void {
     this.validators.push(
-      async (args: CommandArgs) => validation.isValidSharePointUrl(args.options.webUrl)
-    );
+      async (args: CommandArgs) => {
+        const isValidSharePointUrl = validation.isValidSharePointUrl(args.options.webUrl);
+        if (isValidSharePointUrl !== true) {
+          return isValidSharePointUrl;
+        }
+
+        if (args.options.color !== undefined) {
```

Comment:
Currently in the error message, we are only showing either the string values or number values. let's just show every possible valid value always.